spec/factories/task_managements.rb

```diff
@@ -1,10 +1,15 @@
 FactoryGirl.define do
   factory :task_management do
-    start_time Time.now
-    end_time 2.days.from_now
     task_id 1
     tasker_id 1
     taskee_id 2
-    amount 5000
+    task_desc Faker::Lorem.sentence
+    amount "9000"
```

Comment:
Consider using Faker::Commerce.price, https://github.com/stympy/faker#fakercommerce. There are a lot of methods you can leverage for generic parameters. Also, Enclose you call to to Faker with a {} so that it generates a new instance of the parameter every time the code calls it. Something like { Faker::Address.state }.